rfp-proposals/rfp-proposal-storage-helper.md

```diff
@@ -0,0 +1,142 @@
+# RFP Proposal: `Storage Helper Libraries`
+
+**Name of Project: Storage Helper Libraries
+
+**Link to RFP:** [Storage Helper Libraries](https://github.com/filecoin-project/devgrants/blob/master/rfps/rfp-storage-helper-libraries.md)
+
+**RFP Category:** `devtools-libraries`
+
+**Proposer:** [@deaswang](https://github.com/deaswang)
+
+**Do you agree to open source all work you do on behalf of this RFP and dual-license under MIT and APACHE2 licenses?:** "Yes"
+
+# Project Description
+
+go-filecoin is to handle operations at the detailed, protocol level. Storage Helper Libraries separate from the go-filecoin node but generally reusable across a variety of storage use cases and implementations.
```

Comment:
Could this project work for lotus as well?

Reply:
yes. it should depend on go-sectorbuilder.